cmd/podman/spec.go

```diff
@@ -351,6 +351,22 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 		configSpec.Linux.Seccomp = nil
 	}
 
+	// Add Secret Mounts
+	mountsFiles := []string{libpod.OverrideMountsFile, config.DefaultMountsFile}
+	for _, file := range mountsFiles {
```

Comment:
Shouldn't we only use one of these, instead of mounting the contents of both?

Reply:
So we first mount what is in the override file, and then mount the rest of the defaults. If the user has the same mount destination in the override and default, only the override one is mounted.

Reply:
/usr/share/containers/mounts.conf is for the distribution to provide defaults.  
/etc/containers/mounts.conf is for the admin to add his own.